[2025-05-20 14:52:02,084] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: ETLWeatherPostgresAndPrint.load manual__2025-05-20T14:51:59.143497+00:00 [queued]>
[2025-05-20 14:52:02,093] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: ETLWeatherPostgresAndPrint.load manual__2025-05-20T14:51:59.143497+00:00 [queued]>
[2025-05-20 14:52:02,093] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2025-05-20 14:52:02,093] {taskinstance.py:1244} INFO - Starting attempt 1 of 1
[2025-05-20 14:52:02,093] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2025-05-20 14:52:02,101] {taskinstance.py:1264} INFO - Executing <Task(_PythonDecoratedOperator): load> on 2025-05-20 14:51:59.143497+00:00
[2025-05-20 14:52:02,106] {standard_task_runner.py:52} INFO - Started process 2128 to run task
[2025-05-20 14:52:02,109] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'ETLWeatherPostgresAndPrint', 'load', 'manual__2025-05-20T14:51:59.143497+00:00', '--job-id', '60', '--raw', '--subdir', 'DAGS_FOLDER/03-ETLWeatherPostgresAndPrint.py', '--cfg-path', '/tmp/tmpz4xs65z0', '--error-file', '/tmp/tmpyg1cdoyy']
[2025-05-20 14:52:02,109] {standard_task_runner.py:77} INFO - Job 60: Subtask load
[2025-05-20 14:52:02,149] {logging_mixin.py:109} INFO - Running <TaskInstance: ETLWeatherPostgresAndPrint.load manual__2025-05-20T14:51:59.143497+00:00 [running]> on host fabc4724d39b
[2025-05-20 14:52:02,196] {taskinstance.py:1431} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=ETLWeatherPostgresAndPrint
AIRFLOW_CTX_TASK_ID=load
AIRFLOW_CTX_EXECUTION_DATE=2025-05-20T14:51:59.143497+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2025-05-20T14:51:59.143497+00:00
[2025-05-20 14:52:02,201] {logging_mixin.py:109} INFO - 1 Record inserted successfully into table
[2025-05-20 14:52:02,201] {logging_mixin.py:109} INFO - PostgreSQL connection is closed
[2025-05-20 14:52:02,201] {python.py:175} INFO - Done. Returned value was: None
[2025-05-20 14:52:02,210] {taskinstance.py:1282} INFO - Marking task as SUCCESS. dag_id=ETLWeatherPostgresAndPrint, task_id=load, execution_date=20250520T145159, start_date=20250520T145202, end_date=20250520T145202
[2025-05-20 14:52:02,240] {local_task_job.py:154} INFO - Task exited with return code 0
[2025-05-20 14:52:02,265] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
